spring-boot-project/spring-boot-tools/spring-boot-test-support-docker/src/main/java/org/springframework/boot/testsupport/container/TestImage.java

```diff
@@ -90,6 +92,12 @@ public enum TestImage {
 			(container) -> ((CouchbaseContainer) container).withStartupAttempts(5)
 				.withStartupTimeout(Duration.ofMinutes(10))),
 
+	/**
+	 * A container image suitable for testing DB2.
+	 */
+	DB2("ibmcom/db2", "11.5.8.0", () -> Db2Container.class,
```

Comment:
https://hub.docker.com/r/ibmcom/db2 doesn't look like it's going to be updated in the future.

Reply:
The successor `icr.io/db2_community/db2` is supported also, but there is no page list tags, using `latest` is not a good option.

Reply:
OK, I changed it to `icr.io/db2_community/db2`